### PR TITLE
Updates to a few Go rules

### PR DESCRIPTION
**Pull request by 0xDC0DE**

### CODE REVIEW

**Review by inkz — APPROVED** [review]

**Review by inkz — DISMISSED** [review]
all good! :+1:  but looks like there are errors for bot metadata checks and general rules tests

File: go/lang/security/audit/crypto/bad_imports.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 rules:
 - id: insecure-module-used
   message: >-
-    Detected use of an insecure cryptographic hashing method. This method is known
-    to be broken and easily compromised. Use SHA256 or SHA3 instead.
+    The package `net/http/cgi` is on the import blocklist. 
+    The package is vulnerable to httpoxy attacks (CVE-2015-5386).
```
Comment (colleend) [inline go/lang/security/audit/crypto/bad_imports.yaml:5]: could we add a recommendation of a fix 🙏

File: go/lang/security/audit/crypto/missing-ssl-minversion.yaml [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 - id: missing-ssl-minversion
   message: >-
-    `MinVersion` is missing from this TLS configuration. The default
-    value is TLS1.0 which is considered insecure. Explicitly set the
-    `MinVersion` to a secure version of TLS, such as `VersionTLS13`.
+    `MinVersion` is missing from this TLS configuration. 
+    By default, TLS 1.2 is currently used as the minimum when acting as a client, and TLS 1.0 when acting as a server.
+    General purpose web applications should default to TLS 1.3 with all other protocols disabled. 
+    Only where it is known that a web server must support legacy clients
+    with unsupported an insecure browsers (such as Internet Explorer 10), it may be necessary to enable TLS 1.0 to provide support.
```
Comment (colleend) [inline go/lang/security/audit/crypto/missing-ssl-minversion.yaml:8]: add a line to say 'explicitly set `minversion` to `versionTLS13`?

**Review by colleend — CHANGES_REQUESTED** [review]
Just added a few nits for the messages.

Thank you for doing this work and for adding in the fixes for alot of these rules 🙏

**Comment by 0xDC0DE** [comment]
Thanks @colleend, I made the requested changes.

**Review by colleend — APPROVED** [review]
awesome work!!

**Review by enncoded — APPROVED** [review]

**Pull request merged**